### PR TITLE
Add missing nullptr check

### PR DESCRIPTION
**Pull request by adrian-prantl**

### CODE REVIEW

**Comment by adrian-prantl** [comment]
@swift-ci test

File: lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp [hunk 1]
```diff
@@ -1178,6 +1178,8 @@ bool lldb_private::formatters::swift::SIMDVector_SummaryProvider(
       return false;
 
     auto synthetic = simd_elem->GetSyntheticValue();
+    if (!simd_elem)
```
Comment (augusto2112) [inline lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp]: This is after the access? Or is this supposed to check `synthetic`?

**Review by augusto2112 — CHANGES_REQUESTED** [review]

File: lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp (hunk 1 above)
Comment (adrian-prantl) [reply lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp]: Good thing we do code reviews!

**Comment by adrian-prantl** [comment]
@swift-ci test

**Comment by adrian-prantl** [comment]
@swift-ci test windows

**Review by augusto2112 — APPROVED** [review]

**Pull request merged**